L/LibCURL/common.jl

```diff
@@ -34,11 +35,22 @@ function build_libcurl(ARGS, name::String, version::VersionNumber)
     # Collection of sources required to build LibCURL
     sources = [
         ArchiveSource("https://curl.se/download/curl-$(version).tar.gz", hash),
+        ArchiveSource("https://github.com/phracker/MacOSX-SDKs/releases/download/10.15/MacOSX10.13.sdk.tar.xz",
+                      "a3a077385205039a7c6f9e2c98ecdf2a720b2a819da715e03e0630c75782c1e4"),
         DirectorySource("../patches"),
     ]
 
     # Bash recipe for building across all platforms
     script = "THIS_IS_CURL=$(this_is_curl_jll)\n" * raw"""
+    if [[ "${target}" == x86_64-apple-darwin* ]]; then
```

Comment:
Is this not needed for `aarch64-apple-darwin`?

Reply:
On aarch64-darwin we use the 11.0 SDK by default, so presumably not

Reply:
But this change should be version-dependent